connectors/connectorLibs/ContentScript.js

```diff
@@ -361,11 +361,26 @@ export default class ContentScript {
   /**
    * Send log message to the launcher
    *
+   * @param {string} : the log level
```

Comment:
* @param {"debug"|"info"|"warn"|"error"} level: the log level

Reply:
Updated